.gitignore

```diff
@@ -142,4 +142,10 @@ notify.json
 .pytype/
 
 # Cython debug symbols
-cython_debug/
\ No newline at end of file
+cython_debug/
+
+# User-Specific Files
+*.json
+*.code-workspace
```

Comment:
Code workspace is a good add but generally we want to keep .json, requirements and bot.py